packages/jaeger-ui/config-overrides.js

```diff
@@ -16,6 +16,8 @@
 const fs = require('fs');
 const { addBabelPlugin, addLessLoader } = require('customize-cra');
 const lessToJs = require('less-vars-to-js');
+const ManifestPlugin = require('webpack-manifest-plugin');
```

Comment:
I am not familiar with the webpack ecosystem, but isn't it possible to configure this plugin via config file, rather than with explicit code like that?

Reply:
https://www.npmjs.com/package/webpack-manifest-plugin

Reply:
Everything I know about it I learned today. So this is probably the best I've got at the moment.